qa/integration-tests/src/test/java/io/camunda/it/StandaloneBackupManagerTest.java

```diff
@@ -0,0 +1,446 @@
+            });
+    return snapshots;
+  }
+
+  /**
+   * Asserts that the data is present in Operate and Tasklist.
+   *
+   * <ul>
+   *   <li>Asserts that the process instance is present in Operate and verifies the check.
+   *   <li>Asserts that one user task is present in Tasklist and returns its key.
+   * </ul>
+   */
+  private long assertThatDataIsPresent(
+      final long processInstanceKey, final Predicate<ProcessInstance> processInstanceCheck) {
+    try (final var operateClient = camunda.newOperateClient()) {
+      Awaitility.await("should find a process instance")
+          .atMost(ofSeconds(30))
+          .ignoreExceptions()
+          .untilAsserted(
+              () -> {
+                final var result =
+                    operateClient
+                        .withAuthentication(TEST_USER_NAME, TEST_USER_PASSWORD)
+                        .getProcessInstanceWith(processInstanceKey);
+                EitherAssert.assertThat(result)
+                    .right()
+                    .isNotNull()
+                    .extracting(ProcessInstanceResult::total)
+                    .isEqualTo(1L);
+                final var processInstance = result.get().processInstances().getFirst();
+                assertThat(processInstance).matches(processInstanceCheck);
+              });
+    }
+    final var userTaskKey = new AtomicReference<Long>();
+    try (final var tasklistClient = camunda.newTasklistClient()) {
+      Awaitility.await("should find a user task")
+          .atMost(ofSeconds(30))
+          .ignoreExceptions()
+          .untilAsserted(
+              () -> {
+                final List<SearchQueryHit<TaskEntity>> hits =
+                    tasklistClient
+                        .withAuthentication(TEST_USER_NAME, TEST_USER_PASSWORD)
+                        .searchUserTasks(SearchQueryBuilders.query().build())
+                        .hits();
+                assertThat(hits).hasSize(1);
+                userTaskKey.set(hits.get(0).source().getKey());
+              });
+    }
+    return userTaskKey.get();
+  }
+
+  private void deleteIndices() throws IOException {
+    adminElasticsearchClient
+        .indices()
+        .delete(r -> r.index("operate*", "tasklist*", "zeebe*").ignoreUnavailable(true));
+    Awaitility.await("should delete indices")
+        .atMost(ofSeconds(30))
+        .pollDelay(ofSeconds(2))
+        .untilAsserted(
+            () ->
+                assertThat(
+                        adminElasticsearchClient
+                            .indices()
+                            .exists(
+                                r ->
+                                    r.index("operate*", "tasklist*", "zeebe*")
+                                        .allowNoIndices(false))
+                            .value())
+                    .isFalse());
+  }
+
+  private void restoreBackup(final List<String> snapshots) {
```

Comment:
🔧 We could link here also to the docs https://docs.camunda.io/docs/self-managed/operational-guides/backup-restore/backup-and-restore/#restore